assets/pages/regular-contributions/components/directDebitContributionsThankYou.jsx

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import SimpleHeader from 'components/headers/simpleHeader/simpleHeader';
+import Footer from 'components/footer/footer';
+import ThankYouIntroduction from 'components/thankYouIntroduction/thankYouIntroduction';
+import QuestionsContact from 'components/questionsContact/questionsContact';
+import SpreadTheWord from 'components/spreadTheWord/spreadTheWord';
+
+import EmailConfirmation from './emailConfirmation';
+import MarketingConsentContainer from './marketingConsentContainer';
+import DirectDebitPaymentMethodDetails from './directDebitPaymentMethodDetails';
+
+export default function DirectDebitContributionsThankYouPage(props: Object) {
+  return (
+    <div className="gu-content">
+      <SimpleHeader />
+      <ThankYouIntroduction
+        highlights={['Thank you']}
+        headings={['for a valuable', 'contribution']}
+      />
+      <div className="multiline-divider" />
+      <EmailConfirmation />
+      <DirectDebitPaymentMethodDetails
+        accountHolderName={props.accountHolderName}
+        sortCodeArray={props.sortCodeArray}
+        accountNumber={props.accountNumber}
+      />
+      <MarketingConsentContainer />
+      <QuestionsContact />
+      <SpreadTheWord />
+      <Footer />
+    </div>
+  );
+}
```

Comment:
There's quite a lot of duplication between this and the `defaultContributionsThankYou` page. I'm aware that this can be a good thing, as it prevents people only being aware of one when making changes, and breaking the other as a result.

However, if you did want to merge them into one you could do it by turning the page into a higher-order component, and passing in the DD component as a prop or as children. This is currently how the `Contribute` component [works](https://github.com/guardian/support-frontend/blob/master/assets/components/contribute/contribute.jsx).

Reply:
I'd be happy to refactor this once we've added the direct debit logo and guarantee to the direct debit thank-you page.